portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,16 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Isabelle's Portfolio</h1>
+      <p>
+          I'm Isabelle, and I'm a rising 3rd year student majoring in Computer Science at the
+          <br>Georgia Institute of Technology! Currently, I am interning at Google as a STEP Intern!
```

Comment:
nit: instead of using `<br>` to insert line breaks, prefer to allow the text to auto-wrap in its container

e.g. you can set the width of the container with `<div id="content" style="max-width: 80em">...</div>` or in CSS with:

```
#content {
  max-width: 80em;
}
```

(`em` is a unit relative to the font size, you may want to play around with it a bit until it looks good)

The main benefit of allowing the text to auto-wrap is that it will look good on all screen sizes; with explicit breaks, a small screen might render the above like:

> I'm Isabelle, and I'm a rising 3rd year student majoring in Computer
> Science at the
> Georgia Institute of Technology! Currently, I am interning at Google
> as a STEP Intern!
> This is my portfolio!